cmd/soroban-rpc/main.go

```diff
@@ -23,6 +23,7 @@ func main() {
 	var endpoint, horizonURL, stellarCoreURL, binaryPath, configPath, networkPassphrase, dbPath, captivecoreStoragePath string
 	var captiveCoreHTTPPort, ledgerEntryStorageTimeoutMinutes uint
 	var checkpointFrequency uint32
+	var useDb bool
```

Comment:
please change `useDb` -> `useDB`

Reply:
updated, I ran the `make lint-changes` locally, but it emits nothing in response, it should have shown this right? I probably have local config problem, installed golangci-lint from brew on mac.

Reply:
I'm not sure if the `lint-changes` works well after you've committed the changes. however, you should be able to run `make lint`, which should not generate any more warnings.